docs/UserGuide.md

```diff
@@ -310,156 +310,156 @@ the constraints of each parameter when used in a command.
 
 Adds a patient and their relevant details to MediBase3.
 
-Format: `add n/NAME i/NRIC g/GENDER d/DOB p/PHONE_NUMBER e/EMAIL a/ADDRESS`
+**Format**: `add n/NAME i/NRIC g/GENDER d/DOB p/PHONE_NUMBER e/EMAIL a/ADDRESS`
 
 {: .alert .alert-info}
 > :information_source: **Note:**
 > 
 > * All fields are compulsory and must be non-empty.
-> * A patient will not be added if the NRIC given is already associated with another patient in MediBase3. An error message will be displayed in this case.
+> * If the `NRIC` provided already exists in MediBase3, an error message will display, and the patient will not be added.
```

Comment:
Should change to 

' If the `NRIC` provided already exists in MediBase3, an error message will  _be displayed_, and the patient will not be added.'